spring-batch-docs/asciidoc/retry.adoc

```diff
@@ -301,27 +304,28 @@ There is also a more flexible implementation called
     user to configure different retry behavior for an arbitrary set of
     exception types though the `ExceptionClassifier`
     abstraction. The policy works by calling on the classifier to convert an
-    exception into a delegate `RetryPolicy`, so for
+    exception into a delegate `RetryPolicy`. For
     example, one exception type can be retried more times before failure than
     another by mapping it to a different policy.
 
 Users might need to implement their own retry policies for more
-    customized decisions. For instance, if there is a well-known,
-    solution-specific, classification of exceptions into retryable and not
+    customized decisions. For instance, a custom retry policy makes sense when there is a well-known,
+    solution-specific classification of exceptions into retryable and not
     retryable.
 
 [[backoffPolicies]]
 
 
 === Backoff Policies
 
-When retrying after a transient failure it often helps to wait a bit
+When retrying after a transient failure, it often helps to wait a bit
     before trying again, because usually the failure is caused by some problem
-    that will only be resolved by waiting. If a
+    that can only be resolved by waiting. If a
     `RetryCallback` fails, the
     `RetryTemplate` can pause execution according to the
-    `BackoffPolicy` in place.
+    `BackoffPolicy`.
 
+The following code shows the interface definition for the `BackOffPolicy` interface:
```

Comment:
I'd use `implementation` instead of `definition`.

Reply:
Please do. Thanks.

Reply:
After re-reading this.  `definition` is the perfect word here.

Reply:
OK. Switch it back to definition, please. Thanks for re-reading. :)